test/stripe.spec.js

```diff
@@ -308,61 +311,77 @@ describe('Stripe Module', function() {
     describe('when given a non-object variable', () => {
       it('should throw an error', () => {
         expect(() => {
-          stripe.setAppInfo('foo');
+          Stripe('sk_test', {
+            appInfo: 'foo',
+          });
         }).to.throw(/AppInfo must be an object./);
       });
     });
 
     describe('when given an object with no `name`', () => {
       it('should throw an error', () => {
         expect(() => {
-          stripe.setAppInfo({});
+          Stripe('sk_test', {
+            appInfo: {},
+          });
         }).to.throw(/AppInfo.name is required/);
 
         expect(() => {
-          stripe.setAppInfo({
-            version: '1.2.3',
+          Stripe('sk_test', {
+            appInfo: {
+              version: '1.2.3',
+            },
           });
         }).to.throw(/AppInfo.name is required/);
 
         expect(() => {
-          stripe.setAppInfo({
-            cats: '42',
+          Stripe('sk_test', {
+            appInfo: {
+              cats: '42',
+            },
           });
         }).to.throw(/AppInfo.name is required/);
       });
     });
 
     describe('when given at least a `name`', () => {
       it('should set name, partner ID, url, and version of stripe._appInfo', () => {
-        stripe.setAppInfo({
-          name: 'MyAwesomeApp',
+        let stripe = Stripe('sk_test', {
+          appInfo: {
+            name: 'MyAwesomeApp',
+          },
         });
         expect(stripe._appInfo).to.eql({
           name: 'MyAwesomeApp',
         });
 
-        stripe.setAppInfo({
-          name: 'MyAwesomeApp',
-          version: '1.2.345',
+        stripe = Stripe('sk_test', {
+          appInfo: {
+            name: 'MyAwesomeApp',
+            version: '1.2.345',
+          },
         });
         expect(stripe._appInfo).to.eql({
           name: 'MyAwesomeApp',
           version: '1.2.345',
         });
 
-        stripe.setAppInfo({
```

Comment:
Let's try to remove the global from this file as well.

Reply:
Decided offline to follow convention in other existing tests in this file of using the name `newStripe` for local stripe instances.